Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -1026,7 +1027,20 @@ extension PackageInfo {
             settingsDictionary["SWIFT_VERSION"] = .string(swiftLanguageVersion)
         }
 
-        return settingsDictionary.isEmpty ? nil : .settings(base: settingsDictionary)
+        if let buildConfigs = buildConfigs,
+           buildConfigs.firstIndex(where: { !["Debug", "Release"].contains($0.name) }) != nil {
```

Comment:
Why do we need this condition? If a custom config is passed we should take it, regardless of its name, isn't it?

Reply:
Debug and Release wil always be in the generated project by default. Adding them here also breaks almost all tests here. So I choose to add settings if custom config specified by the user.

Reply:
I wouldn't add this change if it's only for the tests. Will it be so hard to fix them without including this line?